examples/amm/src/lib.rs

```diff
@@ -101,11 +101,14 @@ linera service --port $PORT &
 
 ### Using GraphiQL
 
-Type each of these in the GraphiQL interface and substitute the env variables with their actual values that we've defined above.
+Type each of these in the GraphiQL interface and substitute the env variables with their actual
+values that we've defined above.
 
 To properly setup the tokens in the proper chains, we need to do some transfer operations:
 
-- Transfer 50 FUN1 from `$OWNER_AMM` in `$CHAIN_AMM` to `$OWNER_1` in `$CHAIN_1`, so they're in the proper chain
+- Transfer 50 FUN1 from `$OWNER_AMM` in `$CHAIN_AMM` to `$OWNER_1` in `$CHAIN_1`, so they're in the proper chain.
+  Run `echo "http://localhost:8080/chains/$CHAIN_AMM/applications/$FUN1_APP_ID"` to print the URL
+  of the GrahpiQL interface for the FUN1 app. Navigate to that URL and enter:
```

Comment:
typo: GrahpiQL

Reply:
Fixing this in https://github.com/linera-io/linera-protocol/pull/2512.